sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -324,84 +320,107 @@ def __init__(self, **kwargs: Any) -> None:
         self._on_refresh_success: Optional[Callable] = kwargs.pop("on_refresh_success", None)
         self._on_refresh_error: Optional[Callable[[Exception], None]] = kwargs.pop("on_refresh_error", None)
 
-    def refresh(self, **kwargs) -> None:  # pylint: disable=too-many-statements
+    def _configuration_refresh(self, client, headers, **kwargs) -> Tuple[bool, List[ConfigurationSetting]]:
```

Comment:
How come in some functions the types of the parameter are added, and in some they aren't?

Reply:
Any method with an _ is considered private and they aren't required for as per the sdk typing rules. Some get added for typing in other areas. They aren't required in Python so by default I typically don't add them. I can go around and at least add them to things in this PR.